includes/discovery/sensors/entity-sensor.inc.php

```diff
@@ -50,10 +54,14 @@
         }
         if ($entitysensor[$entry['entPhySensorType']] && is_numeric($entry['entPhySensorValue']) && is_numeric($index)) {
             $entPhysicalIndex = $index;
-            $oid              = '.1.3.6.1.2.1.99.1.1.1.4.'.$index;
-            $current          = $entry['entPhySensorValue'];
+            $oid = '.1.3.6.1.2.1.99.1.1.1.4.' . $index;
+            $current = $entry['entPhySensorValue'];
             // ENTITY-SENSOR-MIB::entPhySensorUnitsDisplay.11 = STRING: "C"
-            $descr = ucwords($entity_array[$index]['entPhysicalName']);
+            if ($device['os'] === 'arris-d5') {
+                $descr = ucwords($entity_array[$index]['entPhysicalDescr']);
```

Comment:
Nothing really, but, like the CISCO-ENTITY-SENSOR-MIB it was there. I did a os check to use Descr instead of Name again for not breaking other installs/devices

Reply:
Well, it have the Names:
```
...
ENTITY-MIB::entPhysicalName.101 = STRING: tempSensor 1.
ENTITY-MIB::entPhysicalName.102 = STRING: voltSensor 1.
ENTITY-MIB::entPhysicalName.103 = STRING: voltSensor 2.
...
```

Reply:
Those names don't exist in CISCO mib so it should be safe.